src/components/choice-tabs/ChoiceTabs.scss

```
@@ -139,7 +145,15 @@
   letter-spacing: -0.01em; // tighter tracking for heading feel
   color: var(--home-title-color, #111113);
   line-height: 1.25;
-  white-space: nowrap;
+  // Let the label wrap inside the card content box. Combined with the
+  // parent's `min-width: 0` and `flex-wrap: wrap`, this guarantees that the
+  // title is never clipped by the card's `overflow: hidden`.
+  min-width: 0;
+  // Prefer whole-word breaks, but fall back to anywhere so extremely long
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)